osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -31,6 +31,10 @@ public class ScoreInfo : IHasFiles<ScoreFileInfo>, IHasPrimaryKey, ISoftDelete,
         [JsonProperty("accuracy")]
         [Column(TypeName = "DECIMAL(1,4)")]
         public double Accuracy { get; set; }
+        
+        [JsonProperty("modified_accuracy")]
+        [Column(TypeName = "DECIMAL(1,4)")]
+        public double ModifiedAccuracy { get; set; }
```

Comment:
what is this here for? it's not used and there's also no migrtion for it.

Reply:
I think vinxis added it for his bot. I'll remove it.

Reply:
removed